libraries/python_install_helper.rb

```diff
@@ -31,77 +45,77 @@ def rel_path_to_python_binary(new_resource)
       return "bin/python#{revision}"
```

Comment:
Needs to be an absolute directory. Missing leading /

Reply:
For a function named rel_path...? :)